### PR TITLE
[184916597] Route Invalid Media Errors during checkin/checkout to face recognition log

### PR DESCRIPTION
**Pull request by Talleyrand333**
## Is this a Feature, Chore or Bug?
- [] Feature
- [*] Chore
- [] Bug

## Clearly and concisely describe the feature, chore or bug.
Route Invalid Media Errors during checkin/checkout to face recognition log

## Analysis and design (optional)
Analyse and attach the design documentation

## Solution description
Describe your code changes in detail for reviewers.
Route Invalid Media Errors during checkin/checkout to face recognition log
## Is there a business logic within a doctype?
    - [] Yes
    - [*] No

## Output screenshots (optional)
Post the output screenshots, if a UI is affected or added due to this feature.

## Areas affected and ensured
List out the areas affected by your code changes.
Employee Checkin

## Is there any existing behavior change of other features due to this code change?
Mention Yes or No. If Yes, provide the appropriate explanation.

## Did you test with the following dataset?
- [] Existing Data
- [*] New Data

## Was child table created?
    - [] is attachment required?
        did you test attachment
## Did you delete custom field?
    - [] Yes
    - [] No
        If yes, did you write a delete patch?

## Is patch required?
- [] Yes
- [*] No
    ## Was the patch test?

## Which browser(s) did you use for testing?
  - [*] Chrome
  - [] Safari
  - [] Firefox

### CODE REVIEW

**Comment by mymi14s** [comment]
Where is the changes on v1, mobile?

**Review by mymi14s — CHANGES_REQUESTED** [review]
Where is the changes on v1, mobile

**Comment by Talleyrand333** [comment]
Updated the PR @mymi14s

File: one_fm/api/mobile/face_recognition.py [hunk 1]
```diff
@@ -22,101 +22,113 @@
...
+        req = facial_recognition_pb2.FaceRecognitionRequest(
+            username = frappe.session.user,
+            media_type = "video",
+            media_content = video
+        )
+        # Call service stub and get response
+        res = random.choice(stubs).FaceRecognition(req)
+        
+        if res.verification == "FAILED" and res.data == 'Invalid media content':
+            error_message = f"""
```
Comment (mymi14s) [inline one_fm/api/mobile/face_recognition.py:84]: remove html format

**Pull request merged**